ts/indicators/volume-by-price.src.ts

```diff
@@ -361,11 +361,18 @@ seriesType<Highcharts.VBPIndicator>(
             init: boolean
         ): void {
             var series = this,
-                attr: Highcharts.SVGAttributes = {};
+                inverted = series.chart.inverted,
+                group = series.group as Highcharts.SVGElement,
```

Comment:
`Series.group` is typed as `(Highcharts.SVGElement|undefined)`. By casting this to `SVGElement`, we're ignoring the possibility that it can be undefined, which would cause failure below. The more robust approach would probably be that line 370 read `if (!init && group) {`, then everything is properly guarded.

The only "problem" is that we know that `group` is always defined at this point. It is only before the first render, and after destruction, it may be undefined. Any idea for how we should handle such cases @bre1470 ? It is a recurring pattern.

Reply:
Its more complicated then I expected. 
Maybe we should consider different type or union?

Reply:
The official approach is to initialize everything during the construct cycle. With the refactoring we sometimes do an `void 0 as any` assignment, if the init -> render chain takes care of the property value:
```ts
class Series {
    public group: SVGElement = void 0 as any; // dirty!!!
    public render(): void {
        group = this.renderer.createElement('g');
    }
}
```

IMHO the best solution would be `Object.define` support in all our browsers, so we can make use of getter/setter syntax in classes, which would solve all the hassle by automatically creating an initial instance for a property if not set yet:
```ts
class Series {
    private _group?: SVGElement;
    public get group(): SVGElement {
        if (!this._group) {
           this._group = this.renderer.createElement('g');
        }
        return this._group;
    }
}
```

Reply:
> which would solve all the hassle by automatically creating an initial instance for a property if not set yet

A bit like the `getLabel` function of the tooltip, that creates the label if it isn't already defined. But this would produce a ton of extra code if we were to implement it consistently. So I guess, at least in this case we just need to check for `group`. 

`if (!init && group) {`

There's no doubt it is also the safest, as there is, at least in theory, a possibility that this function can run before initialization or after descruction.